changelog/_unreleased/2021-11-05-add-ignorable-tables-system-dump-option,-ignore-dead_message-by-default-and-use-hex-blob-dump.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Add ignorable-tables system:dump option, ignore dead_message by default and use --hex-blob dump
+author: Joshua Behrens
+author_email: code@joshua-behrens.de
+author_github: @JoshuaBehrens
+---
+# Core
+* Added option `ignore-tables` in `system:dump` to allow better re-usage in non e2e cases
+* Added `dead_message` table as default ignored table on dump `system:dump`
```

Comment:
you added three tables as default, if I see that correctly :slightly_smiling_face: